hrm-domain/hrm-core/contact/sql/contact-update-sql.ts

```diff
@@ -28,7 +28,8 @@ SET "rawJson" = COALESCE("rawJson", '{}'::JSONB)
   || (CASE WHEN $<contactlessTask> IS NOT NULL THEN jsonb_build_object('contactlessTask', $<contactlessTask>::JSONB) ELSE '{}'::JSONB END)
   || (CASE WHEN $<callType> IS NOT NULL THEN jsonb_build_object('callType', $<callType>) ELSE '{}'::JSONB END)
   || (CASE WHEN $<definitionVersion> IS NOT NULL THEN jsonb_build_object('definitionVersion', $<definitionVersion>) ELSE '{}'::JSONB END)
-  || (CASE WHEN $<llmSupportedEntries> IS NOT NULL THEN jsonb_build_object('llmSupportedEntries', $<llmSupportedEntries>::JSONB) ELSE '{}'::JSONB END),
+  || (CASE WHEN $<llmSupportedEntries> IS NOT NULL THEN jsonb_build_object('llmSupportedEntries', $<llmSupportedEntries>::JSONB) ELSE '{}'::JSONB END)
+  || (CASE WHEN $<hangUpBy> IS NOT NULL THEN jsonb_build_object('hangUpBy', $<hangUpBy>) ELSE '{}'::JSONB END),
```

Comment:
Does this sets the `hangUpBy` to `'{ }'` if not privded? Is that intended? :thinking: